ground/src/main/java/com/google/android/ground/persistence/local/room/converter/ConverterExt.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 Google LLC
+ * Copyright 2024 Google LLC
```

Comment:
OSPO guidance is to either leave the original date, or add a date range (2022-2024). We've just been leaving the original copyright date so far.

Reply:
Thank you! Done.